packages/grant-explorer/src/features/discovery/RoundCardStat.tsx

```diff
@@ -9,26 +10,29 @@ type RoundCardStatProps = {
 };
 
 function RoundCardStat(props: RoundCardStatProps) {
-  const matchAmount = BigInt(props.matchAmount);
-  const matchAmountNormalized = matchAmount / 1000000000000000000n;
+  const { approvedApplicationsCount, chainId, matchAmount, token } = props;
+  const matchAmountNormalized = BigInt(matchAmount) / 1000000000000000000n;
```

Comment:
I know it's not related to this PR, but we should take the decimals from the token. in case of USDC it's 6 instead of 18

Reply:
> I know it's not related to this PR, but we should take the decimals from the token. in case of USDC it's 6 instead of 18

Do you know if the graphql api has data for the token decimals? 
Otherwise we would need to query the chain for this data using Wagmi's `useToken`:
- https://wagmi.sh/react/hooks/useToken

I suggest we do this in another ticket.